.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,7 @@
-### Description
+<!-- Please remove these Markdown comments before publishing this PR, since the PR message is used as the commit description -->
```

Comment:
As the contributor probably writes a commit message themselves, it might be worth noting *when* the PR description comes into use. (i guess there's some rebasing and/or squashing occurring in your PR merge workflow?)